extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -91,7 +105,10 @@ export const Condition = 'Condition';
 export const PackageReference = 'PackageReference';
 export const Version = 'Version';
 export const PrivateAssets = 'PrivateAssets';
-
+export const ArtifactReference = 'ArtifactReference';
+export const SuppressMissingDependenciesErrors = 'SuppressMissingDependenciesErrors';
+export const DatabaseVariableLiteralValue = 'DatabaseVariableLiteralValue';
+export const DSP = 'DSP';
 
 // SqlProj File targets
 export const NetCoreTargets = '$(NETCoreTargetsPath)\\Microsoft.Data.Tools.Schema.SqlTasks.targets';
```

Comment:
I know this is not your change, just want to check: does this work for non-Windows platform?